apps/dashboard/src/lib/components/Org/Settings/OrgSettings.svelte

```diff
@@ -115,6 +142,10 @@
   function gotoSetting(pathname) {
     goto(`${$currentOrgPath}/settings${pathname}`);
   }
+
+  onMount(() => {
+    hex = $currentOrg.theme;
+  });
```

Comment:
Why are we setting the theme to hex? 

Is the hex actually needed if we finally set it to `theme`?

Reply:
the hex variable is what acts as the background color of the color picker, that inner circle thingy, so we have to set it to the value of the org theme to also determine if we'll be showing the plus svg or not

Reply:
I mean't we should be storing it directly to `$currentOrg.theme` instead of having a local variable.

but it's fine. I have done some refactoring to the code.